aikau/src/main/resources/alfresco/header/CurrentUserStatus.js

```diff
@@ -205,9 +206,8 @@ define(["dojo/_base/declare",
        * 
        * @instance
        */
-      setStatusRelativeTime: function alfresco_header_CurrentUserStatus__setStatusRelativeTime()
-      {
-         var relativeTime = (this.userStatusTime == "" || this.userStatusTime == null) ? this.message("status.never-updated") : this.getRelativeTime(this.userStatusTime);
+      setStatusRelativeTime: function alfresco_header_CurrentUserStatus__setStatusRelativeTime() {
+         var relativeTime = (this.userStatusTime === "" || !this.userStatusTime) ? this.message("status.never-updated") : this.getRelativeTime(this.userStatusTime);
```

Comment:
```javascript
var relativeTime = (this.userStatusTime === "" || !this.userStatusTime) ? this.message("status.never-updated") : this.getRelativeTime(this.userStatusTime);
```

might be more readable as

```javascript
var relativeTime = this.userStatusTime ? this.getRelativeTime(this.userStatusTime) : this.message("status.never-updated");
```

(if my brain's working right, and that follows the same logic)